include/crm/common/results_compat.h

```diff
@@ -25,6 +25,9 @@ extern "C" {
  *             release.
  */
 
+//! \deprecated Use pcmk_rc_str(pcmk_bz2error2rc(rc)) instead
+const char *bz2_strerror(int rc);
```

Comment:
simply "Do not use" since we don't need to provide that capability (and the new function will be internal)